### ISSUE TITLE
aXe a11y audit tool updates to review other screen sizes

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly#1602]
There are elements that are hidden in certain components for certain viewport sizes. Some of these elements aren't getting checked by the aXe a11y audit tool (because it ignores hidden elements).

Here is an example line that shows how screen size can be configured:
https://github.com/patternfly/patternfly-next/blob/master/test-a11y/index.js#L20

Some questions are:

- What checks do we want to enable to be run manually vs automatically per PR (we might not want to run all checks automatically)? 
- What are the different sizes that we need to include in the tests? @mcoker  could you provide some feedback on this?
- This might be a checklist item to add to the PR template, i.e. if your component shows/hides things for different sizes, manually run the tool for different sizes.

cc @seanforyou23

**Comment by mcoker** [comment]
re: sizes needed in the tests, I think we would just check at all of our breakpoints https://github.com/patternfly/patternfly-next/blob/master/src/patternfly/sass-utilities/scss-variables.scss#L149-L154

**Comment by mcoker** [comment]
cc @jessiehuff

**Comment by nicolethoen** [comment]
@mcoker @jschuler This should be possible now with patternfly-a11y v 4.3.0 :)
